src/views/PVChargeConfig.vue

```diff
@@ -65,6 +65,13 @@
             <template #help> Obere Grenze des Regelbereichs. </template>
           </openwb-base-number-input>
           <hr />
+          <openwb-base-alert subtype="warning" v-if="
+            ($store.state.mqtt['openWB/general/chargemode_config/pv_charging/switch_on_threshold']) - 
+            ($store.state.mqtt['openWB/general/chargemode_config/pv_charging/switch_off_threshold']) < 1400">
+            Der einphasige Mindestladestrom von 6A ergibt eine Ladeleistung von etwa 1,4kW. Ein- und 
+            Ausschaltschwelle sollten daher mindestens 1,4kW auseinander liegen, um ständiges 
+            Ein- und Ausschalten zu vermeiden.
+          </openwb-base-alert>
           <openwb-base-number-input
             title="Einschaltschwelle"
             :min="0"
```

Comment:
Das Minimum kann dynamisch anhand der Abschaltschwelle gesetzt werden, um unsinnige Eingaben zu verhindern:
```suggestion
            :min="
              ($store.state.mqtt['openWB/general/chargemode_config/pv_charging/switch_off_threshold'] + 1400) / 1000
            "
```

Bei der Abschaltschwelle gilt das analog für das Maximum (Zeile 110/111):
```
            :max="($store.state.mqtt['openWB/general/chargemode_config/pv_charging/switch_on_threshold'] - 1400) / 1000"
```